app/components/avatar-image-fallback/avatar-image-fallback.component.ts

```diff
@@ -0,0 +1,65 @@
+import angular from 'angular';
+
+interface AvatarImageFallbackBindings {
+    src?: string;
+    avatarClass?: string;
+    name?: string;
+}
+
+class AvatarImageFallback implements AvatarImageFallbackBindings {
+    // bindings
+    src?: string;
+    avatarClass?: string;
+    name?: string;
+
+    // interface
+    imageClass!: string;
+    charCount!: number;
+
+    constructor(private $element) {}
+
+    $onInit() {
+        this.imageClass = this.avatarClass ? this.avatarClass : 'fallback';
+        this.setCharCount();
+
+        this.$element.find('img').bind('error', () => {
+            const appColors = ['#007BE3', '#FFC715', '#00D6B2', '#FF4848', '#385064'];
+            const defaultFallback = angular
+                .element(document.createElement('i'))
+                .addClass(`fas fa-user app-avatar-image ${this.avatarClass}`)
+                .css({
+                    background: appColors[Math.floor(Math.random() * appColors.length)]
+                });
+            this.$element.parent().prepend(defaultFallback);
+            this.$element.remove();
+        });
+    }
+
+    isProfilePictureShowing() {
+        if (!this.src || this.src.includes('default-user')) {
+            return false;
+        } else {
+            return true;
+        }
+    }
```

Comment:
let's take this logic out - this component should be agnostic to content inside the src. It'll be up to the parent components to handle this.